legal-api/migrations/versions/004f7558063f_second.py

```diff
@@ -977,18 +993,22 @@ def downgrade():
         batch_op.drop_constraint(None, type_='foreignkey')
         batch_op.drop_constraint(None, type_='foreignkey')
         batch_op.drop_constraint(None, type_='foreignkey')
+        batch_op.drop_constraint(None, type_='foreignkey')
         batch_op.drop_index(batch_op.f('ix_offices_legal_entity_id'))
         batch_op.drop_index(batch_op.f('ix_offices_change_filing_id'))
+        batch_op.drop_index(batch_op.f('ix_offices_alternate_name_id'))
         batch_op.drop_column('version')
         batch_op.drop_column('legal_entity_id')
         batch_op.drop_column('change_filing_id')
+        batch_op.drop_column('alternate_name_id')
         batch_op.drop_column('deactivated_date')
         batch_op.drop_column('office_type')
 
     with op.batch_alter_table('legal_entities', schema=None) as batch_op:
         batch_op.drop_constraint(None, type_='foreignkey')
         batch_op.drop_constraint(None, type_='foreignkey')
         batch_op.drop_constraint(None, type_='foreignkey')
+        batch_op.drop_constraint(None, type_='foreignkey')
```

Comment:
This was missing before.

Reply:
thanks for catching this.  Think I forgot to do this when I added the `state_filing_id` FK.